src/events/kafkajs.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * @fileoverview Handlers for kafkajs instrumentation
+ */
+
+const uuid4 = require('uuid4');
+const tracer = require('../tracer.js');
+const eventInterface = require('../event.js');
+const moduleUtils = require('./module_utils.js');
+const { EPSAGON_HEADER } = require('../consts.js');
+
+
+/**
+ * acts as a middleware for `producer.send()`
+ * @param {object} messages the messages param to send
+ * @param {Kafka} producer producer object
+ * @returns {Promise} The response promise
+ */
+function kafkaMiddleware(messages, producer) {
+    let response = Promise.resolve();
```

Comment:
this is not good enough, I am afraid. check out other places we are mocking apis that return a promise and see how to handle it, an empty response is not what the client wants in case of an internal error on our end

Reply:
👍